### ISSUE TITLE
EXCEPTION: Error: Uncaught (in promise): No Pipe decorator found on ApolloQueryPipe

### ISSUE THREAD

**Issue by iguissouma** [apollostack/angular2-apollo#15]
Hello,
I'm getting this error when I run the hello-world example:
`EXCEPTION: Error: Uncaught (in promise): No Pipe decorator found on ApolloQueryPipe`

Any idea how to fix this.
Thanks in advance

**Comment by kamilkisiela** [comment]
@iguissouma Are you using `angular2-apollo` with one of beta versions of `angular2` or with release candidate? It's been tested on beta.

**Comment by iguissouma** [comment]
@kamilkisiela 
I'm using angular2@2.0.0-beta.17, that's what i see on package.jon on node_modules\angular2:

```
"name": "angular2",
  "version": "2.0.0-beta.17",
  "description": "Angular 2 - a web framework for modern web apps",
  "homepage": "https://github.com/angular/angular",
  "bugs": {
    "url": "https://github.com/angular/angular/issues"
  }
```

Do I have to use 2.0.0-beta.15?

**Comment by bharath-cchmc** [comment]
I got this error when I changed the location of the ```pipe.ts``` file.
My folder structure before was client >> 

> pipe.ts, app.ts

when I changed this to 

> client>>pipe>>pipe.ts
> client>>app.ts

 I got the error 
```
EXCEPTION: Error: Uncaught (in promise): No Pipe decorator found on *pipeclass*
```

**Comment by kamilkisiela** [comment]
Could you try to use this #17 ?

**Comment by Urigo** [comment]
Just released 0.1.0 that might solve it, can you try the new version and let me know if it works for you?

**Comment by Urigo** [comment]
closing as I haven't got feedback
